spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationHelper, type: :helper do
+  context 'date_to_string' do
+    it 'formats a date' do
+      expect(date_to_string(Date.current)).to be_a_kind_of String
```

Comment:
I pondered this but in the spirit of having one place where the format was specified, I opted not to do this.  Do you think it's better to have the format string appear twice?  I also considered stubbing strftime, would that make more sense here?